client/src/components/LandingPageHeader/LandingPageHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import { Typography } from '@mui/material';
+import { Box } from '@mui/system';
+
+interface LandingPageHeaderProps {
+  header: string;
+}
+
+const LandingPageHeader: React.FC<LandingPageHeaderProps> = ({ header }) => {
```

Comment:
Good thinking trying to componentize this but since this component is only returning a header, can we not include the relevant code as a part of the parent component?